tests/memory/README.md

```diff
@@ -6,3 +6,4 @@ This directory tests enclave memory management with the following tests:
     and freeing.
   - Stress test the malloc family functions by rapid allocation and freeing
     in a multi-threaded context.
+  - Checking that fragment after mallocing fixed size and random size
```

Comment:
Can you please elaborate on this statement? It is not clear to me exactly what is being checked.